taskcluster/scripts/build/macosx_build.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+# Dependencies
+python3 scripts/tooltool.py --url http://taskcluster/tooltool.mozilla-releng.net fetch -m macos/qt-dependencies.tt.manifest
+export PATH="`pwd`/qt/bin:$PATH"
+export PATH="$HOME/Library/Python/3.6/bin:$HOME/.gem/ruby/2.6.0/bin:$PATH"
+
+# install xcodeproj which is needed by xcode_patcher.rb
+# use --user-install for permissions
+gem install xcodeproj --user-install
+
+# install rust
+curl https://sh.rustup.rs -sSf | sh -s -- -y
```

Comment:
not a fan of piping of piping stuff into the terminal 😅 
Since we have homebrew, could we maybe instead do:
```brew install rust```